doctr/transforms/modules/tensorflow.py

```diff
@@ -459,13 +472,6 @@ def __init__(self, p: float) -> None:
         self.p = p
 
     def __call__(self, img: Union[tf.Tensor, np.ndarray], target: Dict[str, Any]) -> Tuple[tf.Tensor, Dict[str, Any]]:
-        """
```

Comment:
i have removed this because we don't document call methods instead every class has a docstring this was so only one where the call has had a docstring

Reply:
Ok !